lib/settings/subjects.rb

```diff
@@ -7,7 +7,10 @@ class << self
       def each(sorted: false)
         list = Settings::Db.store.subjects
         if sorted
-          list = list.sort_by{ |subject| subject.last['title']}
+          list = list.sort_by do |subject|
+            # force "not listed" to appear at end
+            subject.first == 'not_listed' ? 'zzzz' : subject.last['title']
```

Comment:
It coms from either the seed data in `config/app.yml` or from whatever is set by the admin in `rails-settings-ui`  

As you can gather, this isn't super great since it relies on the key being set to `not_listed` and will break if someone sets it to `not-listed` or something else.

Without a real schema, I think this is about the best we can do though